### DISCUSSION TITLE
Understanding cons for accessing a variable without using Aliases

### DISCUSSION THREAD

**Discussion by burrack** [cypress-io/cypress#16557] [Questions and Help]
Hi,

I saw that the [documentation](https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases) does not recommend using variables for accessing variables that are coming for an asynchronous action.

![image](https://user-images.githubusercontent.com/25310016/118528259-1f743000-b710-11eb-905c-02d83f74bc36.png)

I would like to understand better what are the cons/caveats for the above example?

Thanks!

**Comment by bahmutov** [comment]
Yeah, I agree, there is nothing bad about this, local variables are the simplest way and the most predictable. I wonder how we got to this documentation page

**Reply by burrack** [reply]
Should I create a PR for removing it? 😃
